spec/support/data_api/data_api_spec_helper.rb

```diff
@@ -0,0 +1,23 @@
+module DataAPISpecHelper
+  def get_api_request(url, token:, options: {})
```

Comment:
Since this logic will be used in the register API, is it possible to move this to a more generic helper so we can accommodate both?

Reply:
Yep - if possible let's make it generic. I'll leave that to the Register PR.